src/app/modules/core/core.errors.ts

```diff
@@ -63,8 +64,9 @@ export enum ErrorCodes {
   SUBMISSION_MALICIOUS_FILE_DETECTED = 100220,
   SUBMISSION_INVALID_WORKFLOW_TYPE = 100221,
   SUBMISSION_ATTACHMENT_UPLOAD = 100222,
+  SUBMISSION_EXPECTED_RESPONSE_NOT_FOUND = 100223,
   // Email Submission Errors (email mode deprecated soon)
-  EMAIL_SUBMISSION_HASH = 100223,
```

Comment:
Generally we try to avoid changing error codes. Any specific reason why this is done? 🤔

Reply:
i initially changed the error code for the email mode error so that email mode submission errors will use the code range 100250 - 100299 and for the generic submission errors to follow incrementing order, especially since email mode may be deprecated leaving a gap in the error code order. 

however, perhaps it is fine for the SUBMISSION_EXPECTED_RESPONSE_NOT_FOUND error to skip 1 number and be set as 100224.

Reply:
i initially changed the error code for the email mode error so that email mode submission errors will use the code range 100250 - 100299 and for the generic submission errors to follow incrementing order, especially since email mode may be deprecated leaving a gap in the error code order.

however, perhaps it is fine for the SUBMISSION_EXPECTED_RESPONSE_NOT_FOUND error to skip 1 number and be set as 100224.

Reply:
Yeah ok to skip 1. There's no need for them to be gapless incremental. It just happened to be that way.

We'll generally prefer keeping the old error codes even if they are not utilised as they are already being referenced in the older logs. Updating what the error codes mean (or having multiple meanings to it) makes it hard to traceback in the future. For this scenario we'll prefer to optimise for easier debugging!

Reply:
for logging of rationale: 
Ken: 
Yeah ok to skip 1. There’s no need for them to be gapless incremental. It just happened to be that way.
We’ll generally prefer keeping the old error codes even if they are not utilised as they are already being referenced in the older logs. Updating what the error codes mean (or having multiple meanings to it) makes it hard to traceback in the future. For this scenario we’ll prefer to optimise for easier debugging!